src/style.css

```diff
@@ -0,0 +1,101 @@
+@font-face {
+  font-family: "Avenir-heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-book";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-weight: 300;
+}
+
+.big-form,
+.small-form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 4px;
+
+  font-family: "Avenir-book", sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.input:focus {
+  font-family: "Avenir-heavy", sans-serif;
+  
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(180deg, #fff 0, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.img-big {
+  position: absolute;
+
+  left: 26px;
+  top: 25px;
+  height: 19px;
+  width: 19px;
+}
+
+.input-small {
+  height: 42px;
+  width: 100%;
+  left: 0;
+  top: 0;

```

Comment:
`top` and `left` do nothing because you didn't write position for `input-small`
you can remove `top` and `left`